src/options.rs

```diff
@@ -134,6 +134,13 @@ pub struct BuildOptions {
     /// the `*-trace-compares` instrumentation assumes that the instruction is
     /// available.
     pub no_trace_compares: bool,
+
+    #[arg(long)]
+    /// Disable transformation of if-statements into `cmov` instructions (when this
+    /// happens, we get no coverage feedback for that branch). Default setting is true.
+    /// A further explanation can be found here:
+    /// https://github.com/rust-fuzz/cargo-fuzz/pull/380#issue-2445529059
```

Comment:
Let's inline the explanation here. It will be more likely to be kept up to date that way, and user's won't have to chase hyperlinks.

Reply:
Please see the latest commit; I've condensed it as much as I can, but it's definitely a long doc comment.